### ISSUE TITLE
Issue with replacing &nbsp; with " "

### ISSUE THREAD

**Issue by awunes** [itzTheMeow/nitrotype-instatyper#1]
The &nbsp; replacement seems to be failing. I'm not sure if that's the issue but I think it's this line that's failing:

titleText = titleText.replace(/ /g, " "); // Replaces all the Non Breaking Spaces they use with regular spaces.
between / / is actually: "Â " in Ansi and " " in UTF-8. If I convert to Ansi then view in UTF-8 I see: 
![utf-8 from ansi](https://user-images.githubusercontent.com/31239393/138336746-2638e683-eef1-404e-b721-8d75e8f54731.JPG)

I'm using notepad++ to see this But I can't find any information on how to correct it in the .user.js "tampermonkey script"! Can you help?

**Comment by itzTheMeow** [comment]
i'm pretty sure nt updated their site because i tried the script a while ago and it didn't seem to work properly. i'm going to need to look at the text detection and fix it

`\xa0` is a nbsp in javascript if you want to try that, although they probably changed their spacing

**Comment by awunes** [comment]
Yeah, I was thinking about trying to make this kind of script in AHK and knew of "greasemonkey" from years ago (about 6-10years) so I was like "This can be done but how?" then I decided to get "tampermonkey" and searched for nitrotype.com and found this and was like "WOW!!! exactly what I was looking for!" Just didn't want to work too well :(  Anyway I'm about to try the character you suggested.

Would it look like this then? 
`titleText = titleText.replace(/\xa0/g, " ")`
The issue I'm having is I"m not sure how the .replace system/syntex works.
Okay so I tested it just now and it worked! However it typed @ 425wpm! That's a bit too fast and will definitely get someone banned.
